src/klein/_headers.py

```diff
@@ -143,8 +138,8 @@ def rawHeaderNameAndValue(name, value):
             )
         return (name, value)
 
-    elif isinstance(name, Text):
-        if not isinstance(value, Text):
+    elif isinstance(name, str):
+        if not isinstance(value, str):
             raise TypeError(
                 "value {!r} must be text to match name {!r}".format(value, name)
```

Comment:
Should this be changed to `must be str to match name`?

Reply:
yes